### ISSUE TITLE
Travis doesn't build

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/haxe.org#124]
```
Build tasks.hxml
/home/travis/haxelib/ufront-mvc/1,0,0-rc,12/src/ufront/web/context/HttpContext.hx:13: characters 7-34 : Class not found : thx.core.error.NullArgument
Build server.hxml
/home/travis/haxelib/ufront-mvc/1,0,0-rc,12/src/ufront/web/context/HttpContext.hx:13: characters 7-34 : Class not found : thx.core.error.NullArgument
Build client.hxml
Failed to build tasks.hxml, server.hxml
```

@jasononeil @fponticelli

**Comment by jasononeil** [comment]
That'll be this one: https://github.com/fponticelli/thx.core/pull/46

Fixing it up now.

**Comment by jasononeil** [comment]
Fixed by 944f90b990b678349f33a2517d7006f2ee00ee66
